docker/ubuntu-based/dev/Dockerfile

```diff
@@ -30,9 +30,7 @@ RUN /bin/bash -c "if [ '${VPP_COMMIT_ID}' != 'xxx' ]; then \
         git checkout ${VPP_COMMIT_ID} && \
         rm -rf build-root/ && \
         git reset --hard HEAD && \
-        yes | make install-dep && \
-        make bootstrap && \
-        make pkg-deb; \
+        UNATTENDED=y make vpp_configure_args_vpp='--disable-japi --disable-vom' install-dep bootstrap pkg-deb; \
```

Comment:
you you sure you can build contiv-vpp image with it ? i tried on docker-vpp and it doesn't work

Reply:
What didn't work? I've tried these changes already.

Reply:
when compile contiv-vpp it needs vpp headers, i saw things failed when added those `--disable-xx`
could you please try update `docker-vpp` and try compile `contiv-vpp` with https://github.com/contiv/vpp/tree/master/docker/vpp-vswitch ?

Reply:
@tiewei I replied to the issue why it didn't work for you. You added extra parameter from what I initially suggested. https://github.com/contiv/docker-vpp/issues/3